docs/source/how_to/how_to_algorithm_selection.ipynb

```diff
@@ -113,26 +206,73 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "res = om.minimize(\n",
-    "    fun=sphere,\n",
-    "    params=start_params,\n",
-    "    algorithm=\"nag_pybobyqa\",\n",
-    ")\n",
-    "res.n_fun_evals"
+    "fig = om.criterion_plot(results)\n",
+    "fig.show(renderer=\"png\")"
```

Comment:
Not rendered, probably because nlopt is missing?

Reply:
Thanks, fixed!